app/cli-client/src/app/proto/hello_world_service_pb_service.js

```diff
@@ -0,0 +1,52 @@
+// package: com.google.startuposexample.app.localserver.service
+// file: hello_world_service.proto
+
+var hello_world_service_pb = require("./hello_world_service_pb");
+var grpc = require("grpc-web-client").grpc;
+
+var HelloWorldService = (function () {
+  function HelloWorldService() {}
+  HelloWorldService.serviceName = "com.google.startuposexample.app.localserver.service.HelloWorldService";
+  return HelloWorldService;
+}());
+
+HelloWorldService.helloWorld = {
+  methodName: "helloWorld",
+  service: HelloWorldService,
+  requestStream: false,
+  responseStream: false,
+  requestType: hello_world_service_pb.HelloWorldRequest,
+  responseType: hello_world_service_pb.HelloWorldResponse
+};
+
+exports.HelloWorldService = HelloWorldService;
```

Comment:
Did you modify any of the auto-generated files?

Reply:
No